### ISSUE TITLE
Dagit can be super laggy on large pipelines

### ISSUE THREAD

**Issue by schrockn** [dagster-io/dagster#1127]
SC has a couple pipelines that are very large and cause some pathological. The pipeline in question has ~250 solids most of them have large sql statements associated with them

**Comment by mgasner** [comment]
#1557

**Comment by schrockn** [comment]
I believe that @bengotow addresses this issue. The problems are no longer perf, but layout.

**Comment by bengotow** [comment]
Yeah I believe this is the case. We could also get a large perf bump here and in log / run display if we switch to using React in production mode, but that comes with some debuggability implications so we haven't made the switch yet.

**Comment by schrockn** [comment]
Closing this for now with https://dagster.phacility.com/D869
